### ISSUE TITLE
`main` currently broken - sorry all!

### ISSUE THREAD

**Issue by 149-code** [hexops/mach#453]
Cloning the mach repo and trying to build examples doesn't work. This also happens when I try to build other examples as well.
```
git clone https://github.com/hexops/mach/
cd mach/
zig build run-example-triangle
```

```
/src/platform/c.zig:1:15: error: C import failed
pub const c = @cImport({
              ^
./zig-cache/o/4cb64c7e40f0c8be809e30e746611988/cimport.h:3:10: note: 'dawn_native_mach.h' file not found
#include <dawn_native_mach.h>
```

I checked the whole repo with ```find . | rg "dawn_native_mach.h"``` but no files were found.

I'm compiling with zig version 0.10.0-dev.3366+e6b3eae49 on macOS 12.4

**Comment by theoparis** [comment]
I have a similar issue on Arch Linux. I also tried to re-clone mach and then I got 

```
: /theodata/projects/minecraft-mods/elementalcraft/deps/mach/gpu/libs/mach-gpu-dawn/libs/dawn/src/dawn/utils/VulkanBinding.cpp:23:10: fatal error: 'GLFW/glfw3.h' file not found\
```

even though `/usr/include/GLFW/glfw3.h` exists.

**Comment by slimsag** [comment]
Sorry for `main` being broken, y'all!

It was only supposed to be broken for ~24h while I landed the remainder of #403 (had to do this to get some other parts in), but I suddenly caught the flu, have been quite sick since, and then [lost someone dear to me](https://twitter.com/slimsag/status/1557220441873297410?s=20&t=o7OtLMwqGAivrVB6FFJFIg) just yesterday.

In the meantime you can `git checkout 36539d7c3db3e807047d457e65fc3aeb3145bafa` and as long as you have a fairly recent Zig v0.10 version (`0.10.0-dev.3027+0e26c6149`) it should just work 

I'll have it all working again soon-ish.

**Comment by slimsag** [comment]
Should be fixed now that #403 is merged! Feel free to open a new issue if you encounter any problems.

**Comment by slimsag** [comment]
@sentalis no worries, thanks for the report! #454 was just merged which should fix that, we'll start testing with the latest Zig version soon (which is what caused this)
